watertap/unit_models/pressure_exchanger.py

```diff
@@ -376,6 +420,19 @@ def calculate_scaling_factors(self):
             sf = iscale.get_scaling_factor(self.low_pressure_side.properties_in[t].pressure)
             iscale.constraint_scaling_transform(c, sf)
 
+        if hasattr(self, 'eq_mass_transfer_from_high_pressure_side'):
+            for (t,p,j), c in self.eq_mass_transfer_from_high_pressure_side.items():
+                sf = iscale.get_scaling_factor(self.high_pressure_side.properties_in[t].get_material_flow_terms(p,j))
+                iscale.constraint_scaling_transform(c, sf)
+
+        if hasattr(self, 'eq_mass_transfer_term'):
+            for (t,p,j), c in self.eq_mass_transfer_term.items():
+                sf = iscale.get_scaling_factor(self.high_pressure_side.mass_transfer_term[t, p, j])
+                sf = sf*iscale.get_scaling_factor(self.low_pressure_side.mass_transfer_term[t, p, j])
+                iscale.constraint_scaling_transform(c, sf)
```

Comment:
Line 431 should be deleted. This scaling factor should just be based on the mass_transfer_term, not the square of the mass_transfer_term (this is multiplying the scaling factor for the mass_transfer_term for the high and low pressure side together).

Reply:
Fixed this as well.